src/GraphQL/Resolver/Element.php

```diff
@@ -81,5 +75,132 @@ public function resolveProperties($value = null, $args = [], $context, ResolveIn
         return $result;
     }
 
-}
+    /**
+     * @param null $value
+     * @param array $args
+     * @param $context
+     * @param ResolveInfo|null $resolveInfo
+     * @return array
+     * @throws \Exception
+     */
+    public function resolveParent($value = null, $args = [], $context, ResolveInfo $resolveInfo = null)
+    {
+        $element = ElementService::getElementById($this->elementType, $value['id']);
+        if ($element) {
+            $parent = $element->getParent();
+            if ($parent) {
+                return $this->extractSingleElement($parent, $args, $context, $resolveInfo);
+            }
+        }
+        return null;
+    }
+
+    /**
+     * @param null $value
```

Comment:
array

Reply:
Sorry I don’t understand what you mean

Reply:
I believe the phpdoc type hint isn't right. It should be array instead of null.

Reply:
I fixed the hint (btw: `$value` has a default value of null which is not checked in the added methods, but same applies for the existing method `resolveProperties`)